mkFit/PropagationMPlex.icc

```diff
@@ -45,7 +46,7 @@ static inline void helixAtRFromIterative_impl(const Tf& __restrict__ inPar,
       const float ptinv  = 1.f/pt;
       const float pt2inv = ptinv*ptinv;
       //p=0.3Br => r=p/(0.3*B)
-      const float k = inChg(n, 0, 0) * 100.f / (-Config::sol*Config::Bfield);
+      const float k = inChg.ConstAt(n, 0, 0) * 100.f / (-Config::sol*(useParamBfield?Config::BfieldFromZR(inPar.ConstAt(n,2,0),r0):Config::Bfield));    
```

Comment:
I believe we decided that use of Matriplex At() and ConstAt() is discouraged in new code, there's an overload of operator() for that.